wp-content/themes/currentorg/partials/projects-single-held.php

```diff
@@ -62,7 +62,7 @@
             echo '<ul class="project-tags">';
             $terms_count = count( $terms );
             $terms_index = 0;
-            $delimiter = ' | ';
+            $delimiter = ' |';
             foreach ( $terms as $term ) {
                 $terms_index++;
                 printf(
```

Comment:
in this printf, there's a spare space inside the anchor, which causes the term spacing to be wonky:

![Screen Shot 2020-06-04 at 19 21 05](https://user-images.githubusercontent.com/1754187/83819735-a9aabc00-a698-11ea-87fa-248c686313b9.png)

Reply:
Fixed in https://github.com/INN/umbrella-currentorg/pull/144/commits/c8ef9d9a33b574f733aafb1bd58535f9c0a65911